tasks/install_odl_via_rpm_repo.yml

```diff
@@ -2,7 +2,7 @@
 - include: add_odl_yum_repo.yml
 
 - name: Install ODL via RPM repo
-  yum:
+  package:
     name=opendaylight
     state=present
     update_cache=yes
```

Comment:
`update_cache` isn't a param for the new `package` module. I'll remove it in a follow-up patch.